packages/request-client.js/test/index.test.ts

```diff
@@ -2031,10 +2032,19 @@ describe('request-client.js', () => {
     });
 
     describe('allows overriding the default currencies', () => {
+      const currencyManager = new CurrencyManager(CurrencyManager.getDefaultList());
```

Comment:
I think this test was inaccurate. Previously, the `AddressBasedPaymentNetwork` initialize it's own currency manager; hence the request can be created. Suppose that we only have `_TEST` currency in the currency manager.

Reply:
I don't understand what you mean? I don't think this was inacurate, why do you need to add ETH?

Reply:
We need ETH to validate the address.

Reply:
ah got it. It worked before because the default currency manager was instanciated?

Reply:
Yep. So the instantiated one contains `_TEST` currency only, and we have another currency manager that uses default list.